striven-erp/striven-editor
# link button check

Test for #132 

* Added test to check for `#link-menu` and visibility when the function is called (visible first, then hidden)
* Remove extra spaces at the end or in between (cleaner test.js file)
* Run test locally
